R/helpers_knitr_NextBest.R

```diff
@@ -0,0 +1,621 @@
+# Integration with knitr ----
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' We provide additional utility functions to allow human-friendly rendition of
+#' crmPack objects in Markdown and Quarto files.
+#'
+#' @return a character string that represents the object in markdown.
+#' @name knit_print
+NULL
```

Comment:
this should not be here/ only be in a single central place?
e.g. https://github.com/Roche/crmPack/blob/main/R/helpers_knitr_CohortSize.R already has this. maybe could be good to have this in a central `helpers_knitr.R` file.

Reply:
We discussed whether the `knit_print` helpers should be in a single file or many previously.  Separate files was part of the original proposal (my preference), but you wanted to keep them separate.  Apologies, but I can't find the reference...

So far, the `knit_print` methods encompass 1518 statements, with methods for about another 10 classes/superclasses still to implement.  That would make a single file one of the largest in the package, so I've come round to your way of thinking.  It seems you've also come round to mine!

I don't think single file vs many files is a major issue.  we can do it either way.  If we stick with many files, I would prefer to keep the header in each: casual viewers won't know to look in one specific file.

Reply:
Ah, I just meant for this `Integration with knitr` piece. That should not be duplicate. Because otherwise it is not clear which file generates the actual `.Rd` file output.